CDTDatastore/CDTDatastoreManager.h

```diff
@@ -55,9 +55,6 @@ extern NSString *__nonnull const CDTDatastoreErrorDomain;
 
  All datastore files, including attachments and extensions, are deleted.
 
```

Comment:
I think it's something for the change log. Since there is no way to explicitly  close the index manager, you just let it deallocate and clean it's self up.